cmd/oras/push_windows.go

```diff
@@ -0,0 +1,26 @@
+package main

```

Comment:
do we need a `// +build` here too?

Reply:
No need for that. Just like `*_test.go`

See https://golang.org/pkg/go/build/#hdr-Build_Constraints